.eslintrc.js

```diff
@@ -16,7 +16,6 @@ module.exports = {
   extends: [
     "eslint:recommended",
     'plugin:ember/recommended',
-    "standard"
```

Comment:
Why did you remove the linter?

Reply:
the linter also gave errors loading, i didnt spend any time investigating why.  I'll try to dig deeper